cucumber/api/features/step_definitions/policy_load_steps.rb

```diff
@@ -0,0 +1,69 @@
+require 'English'
+
+When(/^I load a large policy with POST$/) do
+  # Generate a large policy with 1000 variables
+  policy_body = "- &my-variables\n" + [*1..1000].map do |ndx|
+    %Q(
+      - !variable
+        id: obj_Safe_00c_0_#{ndx}/password
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/Safe_00c/obj_Safe_00c_0_10
+    )
+  end.join("\n")
+
+  path = '/policies/cucumber/policy/dev/db'
+
+  try_request true do
+    post_json path, policy_body
+  end
+end
+
+When('I use curl to load a policy with special characters and no content type') do
+  policy_body = %Q(
+    - &active-variables
+      - !variable
+        id: AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name/password
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/safeToSync/AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name
+      - !variable
+        id: AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name/username
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/safeToSync/AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name
+      - !variable
+        id: AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name/address
+        annotations:
+          cyberark-vault: 'true'
+          cyberark-vault/accounts: AutomationVault/safeToSync/AccountWithSpecialCharacters~!@#$%^&(){}[]-+=,Name
+  )
+
+  authorization = current_user_credentials.dig(:headers, :authorization)
+
+  url_path = '/policies/cucumber/policy/dev/db'
+  url = URI.parse(Conjur.configuration.appliance_url + url_path)
```

Comment:
This feels like something generally useful: I'd add it into rest_helpers (or anywhere you think is more appropriate) so that here you can just write something like:
```ruby
full_url(url_path)
```